Testing/SystemTests/tests/analysis/PearlPowderDiffractionScriptTest.py

```diff
@@ -20,13 +20,13 @@ def requiredFiles(self):
         filenames = []
 
         # existing calibration files
-        filenames.extend(('PEARL/Calibration/pearl_group_12_1_TT70.cal',
-                          'PEARL/Calibration/pearl_offset_15_3.cal',
-                          'PEARL/Calibration/van_spline_TT70_cycle_15_4.nxs',
-                          'PEARL/Attentuation/PRL112_DC25_10MM_FF.OUT'))
+        filenames.extend(('PEARL/Focus_Test/Calibration/pearl_group_12_1_TT70.cal',
+                          'PEARL/Focus_Test/Calibration/pearl_offset_15_3.cal',
+                          'PEARL/Focus_Test/Calibration/van_spline_TT70_cycle_15_4.nxs',
+                          'PEARL/Focus_Test/Attentuation/PRL112_DC25_10MM_FF.OUT'))
```

Comment:
Good to have the files in different directories, as there are so many input files.